migrations/versions/e495509fad52_add_pdq_field.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: e495509fad52
+Revises: c2622ac44992
+Create Date: 2023-03-09 19:54:29.773619
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e495509fad52'
+down_revision = 'c2622ac44992'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('images', sa.Column('pdq', postgresql.BIT(length=256), nullable=True))
+    op.alter_column('images', 'phash',
+               existing_type=sa.BIGINT(),
+               nullable=True)
+    op.create_index(op.f('ix_images_pdq'), 'images', ['pdq'], unique=False)
+    op.alter_column('images', 'phash', nullable=True)
+
```

Comment:
It looks like there are two `alter_column` operations for the `phash` column. Do we need them both or can one of them be removed?

Reply:
you could probably drop the last one that just makes it nullable.